tokio/src/io/async_fd.rs

```diff
@@ -243,6 +243,18 @@ impl<T: AsRawFd> AsyncFd<T> {
         Self::new_with_handle_and_interest(inner, scheduler::Handle::current(), interest)
     }
 
+    /// Create a new AsyncFd with the provided raw epoll flags for registration.
+    ///
+    /// These flags replace any epoll flags would normally set when registering the fd.
+    #[track_caller]
+    #[cfg(all(target_os = "linux", tokio_unstable))]
```

Comment:
You need to document that this requires unstable. Please add the note we have on all unstable APIs, and double check that the "Available on tokio_unstable only." tag appears in the docs.

Reply:
done